File: src/main/java/com/bajdcc/LALR1/interpret/os/kern/OSTask.java
package com.bajdcc.LALR1.interpret.os.kern;

import com.bajdcc.LALR1.interpret.os.IOSCodePage;
import com.bajdcc.util.ResourceLoader;

/**
 * 【内核】服务
 *
 * @author bajdcc
 */
public class OSTask implements IOSCodePage {
	@Override
	public String getName() {
		return "/kern/task";
	}

	@Override
	public String getCode() {
		return ResourceLoader.INSTANCE.load(getClass());
	}
}

File: src/main/kotlin/com/bajdcc/LALR1/interpret/os/kern/OSTask.kt
package com.bajdcc.LALR1.interpret.os.kern

import com.bajdcc.LALR1.interpret.os.IOSCodePage
import com.bajdcc.util.ResourceLoader

/**
 * 【内核】服务
 *
 * @author bajdcc
 */
class OSTask : IOSCodePage {
    override val name: String
        get() = "/kern/task"

    override val code: String
        get() = ResourceLoader.load(javaClass)
}
